raster/r.univar/r.univar_main.c

```diff
@@ -394,9 +395,9 @@ static void process_raster(univar_stat *stats, thread_workspace *tw,
             zd->bucket.fcells = NULL;
             zd->bucket.dcells = NULL;
         }
-
+    
 #pragma omp for
-        for (int row = 0; row < rows; row++) {
+        for (row = 0; row < rows; row++) {
```

Comment:
This replaces variable in the scope of the for loop by a variable in the scope outside of the parallel block. This does not look right.

Reply:
Needed for MSVC OpenMP. The iterator variable is by default private for an OpenMP for loop.